src/Language/utils.php

```diff
@@ -75,8 +77,7 @@ function isLetter(int $code): bool
  */
 function isNumber(int $code): bool
 {
-    // - or 0-9
-    return $code === 45 || ($code >= 48 && $code <= 57);
+    return $code >= 48 && $code <= 57; // 0-9
```

Comment:
This was a bug which caused the lexer to accept names with `dashes` in them (which it shouldn't).